pages/common/git-reset.md

```diff
@@ -0,0 +1,11 @@
+# git reset
+
+> Undo commits. Reset current HEAD to the specified state.
```

Comment:
Git reset has two common uses: undoing commits, and unstaging changes. Both should be mention in the description.

I'd suggest "Undo commits or unstage changes, by resetting the current git HEAD to the specified state."